cli/pkg/hapi/api.go

```diff
@@ -22,15 +25,32 @@ func (t CliTokenProvider) GetToken() (string, error) {
 	return token.IDToken, nil
 }
 
+type CliAwsCredentialsProvider struct {
```

Comment:
typically in go programs, acronyms are always uppercase. This gets a little weird when types have two accronyms right next to each other

https://stackoverflow.com/questions/39830166/glued-acronyms-and-golang-naming-convention